lib/bloc/user_profile/breez_user_model.dart

```diff
@@ -31,7 +34,9 @@ class BreezUserModel {
         name = json['name'],
         color = json['color'],
         animal = json['animal'],
-        _image = json['image'];
+        _image = json['image'],
+        securityModel = SecurityModel.fromJson(json['securityModel']),
+        waitingForPin = SecurityModel.fromJson(json['securityModel']).pinCode != null;
```

Comment:
why persisting the `watingForPin` ? isn't it used only on start of the app when reading the pin from the keychain?

Reply:
It's used to indicate that the pin was validated. We change it to false after validating pin to start registering notification handlers.

Reply:
I understand. But we don't need to persist it in JSON nor read it from there.